postman-collection/IoT Central.postman_collection.json

```diff
@@ -2483,6 +2521,345 @@
 					"response": []
 				}
 			]
+		},
+		{
+			"name": "File Upload Storage Configuration",
+			"item": [
+				{
+					"name": "Create a storage account",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
+								"key": "Authorization",
+								"value": "Bearer {{token}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n \"kind\": \"BlockBlobStorage\",\r\n \"location\": \"West US\",\r\n \"sku\": \"Premium_LRS\"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://management.azure.com/subscriptions/{{subscriptionId}}/resourceGroups/{{resourceGroupName}}/providers/Microsoft.Storage/storageAccounts/{{accountName}}?api-version=2021-09-01",
+							"protocol": "https",
+							"host": [
+								"management",
+								"azure",
+								"com"
+							],
+							"path": [
+								"subscriptions",
+								"{{subscriptionId}}",
+								"resourceGroups",
+								"{{resourceGroupName}}",
+								"providers",
+								"Microsoft.Storage",
+								"storageAccounts",
+								"{{accountName}}"
+							],
+							"query": [
+								{
+									"key": "api-version",
+									"value": "2021-09-01"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Create a container",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
```

Comment:
This doesn't look right. You need to change the authorization in the UI from "Inherit auth from parent" to "Bearer Token" on the Authorization tab.